source/AI.cpp

```diff
@@ -3652,9 +3652,9 @@ void AI::MovePlayer(Ship &ship, const PlayerInfo &player, Command &activeCommand
 			autoPilot = activeCommands;
 	}
 	bool shouldAutoAim = false;
-	if(Preferences::Has("Automatic aiming") && !command.Turn() && !ship.IsBoarding()
-			&& ((target && target->GetSystem() == ship.GetSystem() && target->IsTargetable())
-				|| ship.GetTargetAsteroid())
+	if((Preferences::AutoAimIndex() ? ((Preferences::AutoAimIndex() == 1) ? (activeCommands.Has(Command::PRIMARY)) : true) : false)
```

Comment:
Using the preferences and activeCommands directly as boolean conditions is likely more clear.
```suggestion
	if((Preferences::AutoAimIndex() == 2 || (Preferences::AutoAimIndex() == 1 && activeCommands.Has(Command::PRIMARY)))
```